uncompyle6/semantics/pysource.py

```diff
@@ -1892,6 +1894,9 @@ def n_fstring_expr(self, node):
         node.conversion = self.FSTRING_CONVERSION_MAP.get(node.data[1].attr, '')
         self.default(node)
 
+    def n_fstring_single(self, node):
+        return self.n_fstring_expr(node)
+
```

Comment:
I think I understand a little bit more about what's going wrong, but it will take me more time to work this out.